#! /bin/sh /usr/share/dpatch/dpatch-run
## no_protocols_by_default.dpatch by  <soren@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set protocols to "none" by default

@DPATCH@
diff -urNad dovecot-1.0.5~/dovecot-example.conf dovecot-1.0.5/dovecot-example.conf
--- dovecot-1.0.5~/dovecot-example.conf	2007-10-08 13:38:23.111439900 +0200
+++ dovecot-1.0.5/dovecot-example.conf	2007-10-08 13:38:41.479672553 +0200
@@ -21,7 +21,7 @@
 # Protocols we want to be serving: imap imaps pop3 pop3s
 # If you only want to use dovecot-auth, you can set this to "none".
 #protocols = imap imaps
-protocols =
+protocols = none
 
 # IP or host address where to listen in for connections. It's not currently
 # possible to specify multiple addresses. "*" listens in all IPv4 interfaces.
